includes/class-wc-taxjar-integration.php

```diff
@@ -435,6 +450,7 @@ public function taxjar_api_call( $options = array() ) {
       $tax_rates = array(
 			  "tax_rate_country" =>   $to_country,
 			  "tax_rate_state" =>     $to_state,
+        "tax_rate_street" =>    $to_street,
```

Comment:
This is causing a database error since the column doesn't exist when creating / updating a tax rate.